internal/pkg/routes/route.go

```diff
@@ -39,31 +48,58 @@ func NewRoute(n, m, s string, log logger.Logger, pub publish.Publisher) Route {
 		subtopic:  s,
 		logger:    log,
 		pub:       pub,
+		messages:  make(chan *nats.Msg, workers),
+		workers:   workers,
```

Comment:
Do you need this worker?

Reply:
Wait... this was mentioned by Go bot. Did he just ignore Go bot remarks?